test/runtests.jl

```diff
@@ -335,7 +335,7 @@ OBJACOORD
 
 FCOORD
 1
-0 0 0 1 -0.9999999999999998
+0 0 0 1 -0.9999999999999999
```

Comment:
@blegat, is it a precise statement that multiplication and division by sqrt(2) is an identity operation on ``Float64``s?